frontend/app/styles/app.css

```diff
@@ -66,3 +74,30 @@ li a {
   background-color: #2988bc !important;
   border-color: #2479a7 !important;
 }
+
+.progress-bar-0 {
+  width: 0% !important;
+  background-color: unset !important;
+}
+.progress-bar-10 {
+  width: 10% !important;
+  background-color: #fc0303 !important;
+}
+
+.progress-bar-25 {
+  width: 25% !important;
+  background-color: #fc0303 !important;
+}
+
+.progress-bar-50 {
+  width: 50% !important;
+  background-color: #fc6703 !important;
+}
+.progress-bar-75 {
+  width: 75% !important;
+  background-color: #fcc603 !important;
+}
+.progress-bar-100 {
+  width: 100% !important;
+  background-color: #4dd100 !important;
+}
```

Comment:
Not very happy with this, but it's safer instead of setting style inline. Wanted to put it in its own stylesheet, but didn't know how to in Ember. Hope that's ok for now.